packages/abstract-signer/src.ts/index.ts

```diff
@@ -101,7 +101,7 @@ export abstract class Signer {
 
     async getTransactionCount(blockTag?: BlockTag): Promise<number> {
```

Comment:
Since `provider.getTransacitonCount` is missing we should remove it from this interface as-well